isso/js/jest-puppeteer.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
This file should be at the root of the repository.
It is currently ignored by Jest when running `npm test`

Reply:
I'd like to keep all javascript-related stuff inside `isso/js/`, so I'll have to figure out a way for Jest to read this file.
See https://github.com/smooth-code/jest-puppeteer#configure-puppeteer

(remains TODO)

Reply:
Point of reference: https://github.com/smooth-code/jest-puppeteer/issues/160#issuecomment-491975158
```js
process.env.JEST_PUPPETEER_CONFIG = require.resolve('./jest-puppeteer.config.js');
```